.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Use Unix line endings for scripts
+*.sh text eol=lf
```

Comment:
I suspect this should be handled with `autocrlf` on your machine right? I wonder what happens if we also set `* text eol=lf`? Would git do the right thing...

Reply:
The problem with `autocrlf` is that it is a machine-wide setting, and I do want `crlf` line endings in most of my repositories. Just not here, since these files end up getting mounted into the Vagrant VMs.

`* text` would force git to treat _all_ files in this repo as text files, which I guess is fine if you're sure you'll never check in binaries in this repo :-).

Based on my trial, it's just the `.sh` files which need to `lf` line endings. For example, the `Dockerfile` files also end up with `crlf` line endings, but `docker` can handle that just fine on Linux.

I don't have a strong preference, so anything that lets me just run `vagrant up` on Windows is fine with me :D.

Reply:
ah i see. This seems fine to me then.